package.json

```diff
@@ -72,7 +73,8 @@
     "showdown": "1.8.1",
     "tmp-promise": "1.0.3",
     "traverse": "0.6.6",
-    "yarn": "1.2.1"
+    "yarn": "1.2.1",
+    "vso-node-api": "6.2.8-preview"
```

Comment:
I assume this implies the API is not stable? But perhaps we don't have any other choice?

Reply:
yes, but it's the simplest was to interact with vsts. :(